scissors/src/main/java/com/lyft/android/scissors/CropView.java

```diff
@@ -94,16 +110,68 @@ private void drawBitmap(Canvas canvas) {
 
-        canvas.drawRect(0, top, left, getHeight() - top, viewportPaint);
-        canvas.drawRect(0, 0, getWidth(), top, viewportPaint);
-        canvas.drawRect(getWidth() - left, top, getWidth(), getHeight() - top, viewportPaint);
-        canvas.drawRect(0, getHeight() - top, getWidth(), getHeight(), viewportPaint);
+        canvas.drawRect(0, top, left, getHeight() - top, viewportPaint); // left
+        canvas.drawRect(0, 0, getWidth(), top, viewportPaint); // top
+        canvas.drawRect(getWidth() - left, top, getWidth(), getHeight() - top, viewportPaint); // right
+        canvas.drawRect(0, getHeight() - top, getWidth(), getHeight(), viewportPaint); // bottom
+    }
+
+    private void drawOvalOverlay(Canvas canvas) {
+        final int viewportWidth = touchManager.getViewportWidth();
+        final int viewportHeight = touchManager.getViewportHeight();
+        final int left = (getWidth() - viewportWidth) / 2;
+        final int top = (getHeight() - viewportHeight) / 2;
+        final int right = getWidth() - left;
+        final int bottom = getHeight() - top;
+        ovalRect.left = left;
+        ovalRect.top = top;
+        ovalRect.right = right;
+        ovalRect.bottom = bottom;
+
+        // top left arc
+        ovalPath.reset();
+        ovalPath.moveTo(left, getHeight() / 2); // middle of the left side of the circle
+        ovalPath.arcTo(ovalRect, 180, 90, false); // draw arc to top
+        ovalPath.lineTo(left, top); // move to top-left corner
+        ovalPath.lineTo(left, getHeight() / 2); // move back to origin
+        ovalPath.close();
+        canvas.drawPath(ovalPath, viewportPaint);
+
+        // top right arc
+        ovalPath.reset();
+        ovalPath.moveTo(getWidth() / 2, top); // middle of the top side of the circle
+        ovalPath.arcTo(ovalRect, 270, 90, false); // draw arc to the right
+        ovalPath.lineTo(right, top); // move to top-right corner
+        ovalPath.lineTo(getWidth() / 2, top); // move back to origin
+        ovalPath.close();
+        canvas.drawPath(ovalPath, viewportPaint);
+
+        // bottom right arc
+        ovalPath.reset();
+        ovalPath.moveTo(right, getHeight() / 2); // middle of the right side of the circle
+        ovalPath.arcTo(ovalRect, 0, 90, false); // draw arc to the bottom
+        ovalPath.lineTo(right, bottom); // move to bottom-right corner
+        ovalPath.lineTo(right, getHeight() / 2); // move back to origin
+        ovalPath.close();
+        canvas.drawPath(ovalPath, viewportPaint);
+
+        // bottom left arc
+        ovalPath.reset();
+        ovalPath.moveTo(getWidth() / 2, bottom); // middle of the bottom side of the circle
+        ovalPath.arcTo(ovalRect, 90, 90, false); // draw arc to the left
+        ovalPath.lineTo(left, bottom); // move to bottom-left corner
+        ovalPath.lineTo(getWidth() / 2, bottom); // move back to origin
+        ovalPath.close();
+        canvas.drawPath(ovalPath, viewportPaint);
```

Comment:
Screenshot added 👍 

Can you provide some more details on `save`/`saveLayer`? I'm happy to give that a try, I'm just not sure how it would be applicable here.